awx/api/views/inventory.py

```diff
@@ -16,17 +16,21 @@
 from rest_framework import status
 
 # AWX
-from awx.main.models import (
-    ActivityStream,
-    Inventory,
-    JobTemplate,
-    Role,
-    User,
-    InstanceGroup,
-    InventoryUpdateEvent,
-    InventoryUpdate,
+from awx.main.models import ActivityStream, Inventory, JobTemplate, Role, User, InstanceGroup, InventoryUpdateEvent, InventoryUpdate
+
+from awx.main.models.label import Label
```

Comment:
The `Label` model is re-imported at the top level of awx.main.models, so it can be put up in the big list above instead of as a separate line here.